homeassistant/components/tibber/sensor.py

```diff
@@ -50,7 +50,7 @@
 SCAN_INTERVAL = timedelta(minutes=1)
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
 PARALLEL_UPDATES = 0
-
+MAX_REQUEST_SPREAD = 20 * 60
```

Comment:
I mean more like, what is the 20 and what is the 60? There's probably a significance on why you note it like this, otherwise it would have been 1200

Reply:
It was to clarify that it is 20 min, but is it better to use 1200?

Reply:
I think if you name the variable `TWENTY_MINUTES` it's clear in the code below (where you use it) what the value is

Reply:
Other option would be to add a variable for MINUTE = 60 and then just do 20 * MINUTE, which will indicate we are talking about 20 minutes